### ISSUE TITLE
vint outputting a warning when no error is found

### ISSUE THREAD

**Issue by pixelastic** [scrooloose/syntastic#1478]
Hello,

I've just installed `vint` (linter for vim), and it correctly warns me of any error in my files. But it also warns me with a `syntastic: warning: checker vim/vint: unrecognized error format` message when I save a clean file.

I'm looking for a way to disable this warning, as the file is clean and I don't want to have to press Enter everytime I save a clean vim file.

### Steps to reproduce

I tried with a simple file with the following content

```vim
echom "test"
```
and 
```vim
echom 'test'
```

The first one correctly gets the warning from Syntastic that I should use single quotes, while the second one display the following error

![vint warning](http://snag.gy/7VDA3.jpg)

### Versions

Using `vint` v0.3.1, `vim` 7.4 and Syntastic 3.6.0 (commit `d353962`).

```vim
let g:syntastic_mode_map={ 'mode': 'active' }
let g:syntastic_vim_checkers = ['vint']
```

### More info

```shell
$ vint test.vim --json
$ echo $?
0
```

With `g:syntastic_debug` set to 3
```
"test.vim" 1L, 13C written
syntastic: 907.557829: &shell = '/home/tim/local/bin/zsh', &shellcmdflag = '-c', &shellpipe = '2>&1| tee', &shellquote = '', &shellredir = '>%s 2>&1', &shellslash = 0, &
shelltemp = 1, &shellxquote = '', &shellxescape = ''
syntastic: 907.558194: UpdateErrors (auto): default checkers
syntastic: 907.558452: CacheErrors: default checkers
syntastic: 907.558974: g:syntastic_aggregate_errors = 1
syntastic: 907.559136: getcwd() = '/home/tim/.oroshi/config/vim'
syntastic: 907.559426: CacheErrors: Invoking checker: vim/vint
syntastic: 907.559880: SyntasticMake: called with options: {'preprocess': 'vint', 'errorformat': '%f:%l:%c:%t: %m', 'makeprg': 'vint test.vim --json', 'returns': [0, 1]}

syntastic: 907.732634: checker output: ['']
syntastic: warning: checker vim/vint: unrecognized error format
syntastic: 907.733011: preprocess: []
syntastic: 907.733226: raw loclist: []
syntastic: 907.733396: getLocList: checker vim/vint returned 0
syntastic: 907.733523: vim/vint raw: []
syntastic: 907.733667: quiet_messages filter: {}
syntastic: 907.733913: aggregated: {'_sorted': 0, '_name': '', '_owner': 2, '_columns': 1, '_rawLoclist': []}
syntastic: 907.734140: sorted: {'_sorted': 1, '_name': '', '_owner': 2, '_columns': 1, '_rawLoclist': []}
Press ENTER or type command to continue
```

Not sure where to go from here, and ready to give any more information you might need.

**Comment by lcd047** [comment]
Pilot error in a recent patch, sorry.  Fixed in aded156.

**Comment by pixelastic** [comment]
Perfect. Thanks!
